includes/admin/plugins.php

```diff
@@ -193,27 +193,45 @@ function perflab_render_plugin_card( array $plugin_data ) {
 		);
 	}
 
-	$details_link = esc_url_raw(
-		add_query_arg(
-			array(
-				'tab'       => 'plugin-information',
-				'plugin'    => $plugin_data['slug'],
-				'TB_iframe' => 'true',
-				'width'     => 600,
-				'height'    => 550,
-			),
-			admin_url( 'plugin-install.php' )
-		)
-	);
+	$title_link_attr = '';
```

Comment:
With my suggestion below accepted, this variable can then be removed because it will never be used.

```suggestion
```